mietechnologies/Columbia-City-Christian-Church-iOS
# Dashboard Screen

## Description



The creation of the Dashboard screen. This will be the screen the users will first see when they open the app, whether they are logged in or not.

This screen will control the majority of navigation for the app. It is a container which will contain the main screens within the app as it is developed.

### Type of Change


## Does this introduce a breaking change?



## How Can This PR Be Tested?



**Test Configuration**:
* Firmware version: 17.5.1
* Hardware: iPhone 15 Pro Max
* Is Simulator:

In the application test the following scenarios:
1) Swipe the main portion (the white space with the text object in the middle) of the screen from left to right.
  * This should update the ScrollingTabView at the top of the screen in accordance with the currently selected view.
2) Scroll to multiple different positions in the ScrollingTabView, tapping on the options throughout.
  * This should update the main view with the relevant screen. At present its just a Text view in the middle with the associated tab title.
3) Select a tab option that is not the "Dashboard" option, then tap on the ChurchButton (in the top-leading corner)
  * This should navigate you back to the "Dashboard" ScrollingTabView option along with updating the main view accordingly

## Checklist:


- [x] Doc strings have been added to any public APIs
- [x] The CHANGELOG.md file has been updated with your changes (1 entry per PR)
- [x] Before merging, rebase your branch with the target branch
